docs/tests/pytest.run.md

```diff
@@ -16,11 +16,15 @@
     * executed t1 test cases.
 
 
-### Run a single test case ###
+### Run a single test file ###
 * ./run_tests.sh -d <dut_name> -n <testbed_name> [-s <list of test cases or files to skip>] -u -c platform_tests/test_link_flap.py
     * execute link flap test case.
     * when specifying test cast list, no need to specify topology with -t.
 
+### Run a single test case ###
+* ./run_tests.sh -d <dut_name> -n <testbed_name> [-s <list of test cases or files to skip>] -u -c qos/test_qos_sai.py -C "testParameter[single_asic]"
```

Comment:
The sample is probably good enough because some test case is wrapped in a class. For this particular case, the correct path should be `qos/test_qos_sai.py::TestQosSai::testParameter`

Reply:
yea, I have run the test with '[single_asic]' and it actually works XD. but yea, I can remove the end part. and it works too.